default.nix

```diff
@@ -0,0 +1,34 @@
+with import <nixpkgs> {};
+
+stdenv.mkDerivation rec {
+  name = "math-comp-book";
+  version = "master";
+  src = ./.;
+
+  nativeBuildInputs = [ texlive.combined.scheme-full ];
+
+  postPatch = ''
+    substituteInPlace tex/Makefile --replace "git describe --tags >> gitrev" ""
```

Comment:
I don't get this, we want the git hash to be in there.

Reply:
If you want to make it work from a non-git checkout, please edit the Makefile in tex and do something like
```
if [ -d .git ]; then git describe --tags >> gitrev; else touch gitrev; fi
```

Reply:
Ok, I'll check if GitHub actions runs it from a git checkout or not

Reply:
They do, but nix would not (it would download a .zip and unfold it somewhere). I think the change makes sense anyway, you are welcome to do it.